tests/storage/moosefs/conftest.py

```diff
@@ -0,0 +1,49 @@
+import pytest
+
+@pytest.fixture(scope='session')
+def host_with_moosefs(host):
+    assert not host.file_exists('/usr/sbin/mount.moosefs'), \
+        "mount.moosefs must not be installed on the host before all tests"
+    host.ssh(['sh', '-c', '"curl https://ppa.moosefs.com/RPM-GPG-KEY-MooseFS > /etc/pki/rpm-gpg/RPM-GPG-KEY-MooseFS"'])
+    host.ssh(['sh', '-c', '"curl http://ppa.moosefs.com/MooseFS-3-el7.repo > /etc/yum.repos.d/MooseFS.repo"'])
+    host.yum_install(['fuse'], save_state=True)
+    host.yum_install(['moosefs-client'])
```

Comment:
You should use this statement instead:
```python
host.yum_install(['fuse', 'moosefs-client'], save_state=True)
```
The saved state must be applied on the two packages.

Reply:
I had some problems with `save_state=True` statement and moosefs-client package.
Basically, the test fails when this option was set to true. I will investigate this topic.

Reply:
Sorry, ignore my previous comment. There are things to improve internally regarding the `yum_install` method, but in your use case the code is ok. :+1:

Reply:
OK,

Thanks!